spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/TaskExecutionControllerTests.java

```diff
@@ -276,10 +277,23 @@ public void testGetExecutionsByNameNotFound() throws Exception {
 	@Test
 	public void testCleanup() throws Exception {
 		mockMvc.perform(delete("/tasks/executions/1")).andExpect(status().is(200));
+		verify(taskLauncher).cleanup("foobar");
+	}
 
+	@Test
```

Comment:
Thanks for your discover on deleting Task Execution causing an error on the job page.   But now I think we need a test for that scenario as well.